doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -77,19 +77,23 @@ def transform_batch(df: pandas.DataFrame) -> pandas.DataFrame:
 # fmt: on
 
 # fmt: off
-# __writing_native_udfs_begin__
+# __writing_default_udfs_tabular_begin__
 import ray
 import pandas as pd
 
 # Load dataset.
 ds = ray.data.read_csv("example://iris.csv")
+ds.default_batch_format()
+# -> pandas.core.frame.DataFrame
 
 # UDF as a function on Pandas DataFrame batches.
 def pandas_transform(df: pd.DataFrame) -> pd.DataFrame:
     # Filter rows.
     df = df[df["variety"] == "Versicolor"]
     # Add derived column.
-    df["normalized.sepal.length"] =  df["sepal.length"] / df["sepal.length"].max()
+    # Notice here that `df["sepal.length"].max()` is only the max value of the column
```

Comment:
How about we call it "batch" or "df_batch" (instead of just "df) to have a stronger indication that it's just a batch?